javascript/react-nesting_navigation/src/App.js

```diff
@@ -1,34 +1,28 @@
 import "./styles.css";
+console.clear();
 
 import avatar from "./img/avatar.jpg";
```

Comment:
You could remove this line and the following one as you don't use it in this file anymore, right?